server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -834,8 +834,9 @@ void sc_osc_handler::tcp_connection::start(sc_osc_handler * self)
 {
     using namespace boost;
     typedef boost::endian::big_int32_t big_int32_t;
+    boost::system::error_code error;
     asio::ip::tcp::no_delay option(true);
-    socket_.set_option(option);
+    socket_.set_option(option, error);
```

Comment:
aeh, no: the funcion has two signatures. the one that was used throws an exception, the one you introduced sets an error code.
there is no benefit, especially, if you don't add error handling code here